portiaui/app/routes/projects/project/spider.js

```diff
@@ -54,5 +54,9 @@ export default Ember.Route.extend({
             this.transitionTo('projects.project',
                 this.modelFor('projects.project'));
         }
+
+        openLinkOptions() {
```

Comment:
can we move this to link-item-list-crawling?
you can use `Ember.inject.service('-routing')` which has the `transitionTo` method there, instead of passing the action around.